docs/source/tutorial/betweenness_centrality.rst

```diff
@@ -0,0 +1,117 @@
+===================================
+Working with Betweenness Centrality
+===================================
+
+Betweenness centrality is the measure of centrality in a graph based on shortest
```

Comment:
Nitpick: It is not "the" measure based on shortest paths, but "a" measure. There are others too, such as closeness, harmonic centrality, as well as many other less frequently used ones.

Reply:
Updated in: https://github.com/Qiskit/retworkx/pull/537/commits/f7e4421111f967d25a0b2303bb34e5e110fdd8cc